book_source/Topics/module4/feature-importance.md

```diff
@@ -52,9 +54,30 @@ This is a technique used to measure the importance of features in a model by eva
 ```
 ```{tab-item} Code
```

Comment:
This needs to be in a tab-set. And, there is no point to have just one tab; you should have multiple tabs. While it is good to commit to your personal branch often, you shouldn't do a Pull Request until you believe the content/code is ready for production.

Reply:
Okay, never mind. The default view presented to me in GitHub apparently doesn't show enough context around the changes and you do have the tab-set. My mistake.